tests/topotests/isis_tilfa_topo1/rt1/isisd.conf

```diff
@@ -16,6 +16,7 @@ interface lo
 interface eth-sw1
  ip router isis 1
  ipv6 router isis 1
+ isis hello-interval 1
```

Comment:
So this is speeding the whole thing up by a factor of 3, and might be more affected by those weird "lockup" issues we've seen in CI. Was this needed for quicker down detection for the LFA test?

Reply:
yes